content/zh/docs/contribute/_index.md

```diff
@@ -1,9 +1,186 @@
 ---
-title: 贡献新内容
-weight: 20
+content_type: concept
+title: 贡献kubernetes文档
```

Comment:
```suggestion
title: 贡献 Kubernetes 文档
```

Reply:
this is an example, there're maybe 10 to 20 places like this

Reply:
thanks for your kindly comment, it's really my problem and I have fixed it.
by the way, translate work take me 2 hour, while git things take  2 days. :)